examples/vertx/README.md

```diff
@@ -0,0 +1,16 @@
+# Containerize a [Eclipse Vert.x](https://vertx.io/) application with Jib
+
+This is an example of how to easily build a Docker image for a [Eclipse Vert.x application](https://vertx.io/) with Jib.
+
+```shell
+./gradlew jibDockerBuild
+
+docker run -d --rm -p 8080:8080 vertx-jib-example
+```
+
+## More information
+
+Learn [more about Jib](https://github.com/GoogleContainerTools/jib).
+Learn [more about Eclipse Vert.x](https://vertx.io).
```

Comment:
nit: Need another newline or a double space after the first line so they don't end up on the same line.